terraform/setup-terraform.sh

```diff
@@ -1,8 +1,23 @@
 S3_BUCKET_NAME=terraform-state-12312d123fa
```

Comment:
You could also use a loop or create a script with the bucket name as input parameter. And call it with different bucket names. Than you do not have this duplication of code

Reply:
I have change the duplication in this push from code Duplication to for loop : https://github.com/davidebschke/Operational-Planning-Tool/pull/17/commits/12a6e8747ee6bece40abad10ff9a0e6fce701fcf